loops_conditions/task3.js

```diff
@@ -0,0 +1,7 @@
+// Завдання 3: Генерація таблиці множення
+const number = 7
+
+for (let i = 1; i < 10; i++){
+    let summma = number * i
```

Comment:
use `const` instead of 'let'

Reply:
`summa` isn't good name, use more informative name like `multipliedNumber`  or just `result`. We can use `sum` when we do addition but here we do multiplication

another alternative name is `product`
The result of a multiplication is called a product.